java/ql/lib/semmle/code/java/dataflow/FlowSteps.qll

```diff
@@ -64,6 +65,20 @@ class AdditionalTaintStep extends Unit {
   abstract predicate step(DataFlow::Node node1, DataFlow::Node node2);
 }
 
+/**
+ * A unit class for adding additional jump steps.
+ *
+ * Extend this class to add additional jump steps that should apply to all
+ * dataflow configurations.
+ */
```

Comment:
What is the difference between a flow/taint step and a jump step?
A flow/taint step keeps the call context, a jump step doesn't?
Should this be explained in the docs in some way or is the following implication assumed to be true:
user uses `AdditionalJumpStep` -> user knows what a jump step is

Reply:
Pretty much, but users don't need to know.